go/chat/storage/storage_blockengine.go

```diff
@@ -12,7 +12,7 @@ import (
 	"golang.org/x/net/context"
 )
 
-const blockIndexVersion = 6
+const blockIndexVersion = 7
```

Comment:
We probably have a lot of random things like that. Is there a way to say something like "delete any cache row that hasn't been read in a month"? Leveldb probably doesn't track things like that?